libs/utils/android/system.py

```diff
@@ -359,7 +359,7 @@ def gfxinfo_get(target, apk_name, out_file):
                     GET_FRAMESTATS_CMD.format(apk_name, out_file))
 
     @staticmethod
-    def monkey(target, apk_name, event_count=1):
+    def monkey(target, apk_name, event_count=1, pct_syskeys=0):
```

Comment:
I definitely agree with you.
Another idea could be to make the monkey manager even more independent by defining his own separate class. It could be useful for several automated testing.

What do you think?